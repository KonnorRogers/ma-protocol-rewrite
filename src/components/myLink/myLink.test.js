import React from "react"
import { render } from "@testing-library/react"
import MyLink from "./myLink.js"

test("Renders without error", () => {
  const tree = render(<MyLink to="/" />)

  expect(tree).toMatchSnapshot()
})
