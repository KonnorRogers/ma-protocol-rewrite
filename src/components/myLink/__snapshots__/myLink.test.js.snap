// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders without error 1`] = `
<DocumentFragment>
  .emotion-0 {
  color: var(--link-blue);
  padding: 0.25rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.emotion-0 {
  color: var(--link-blue);
  padding: 0.25rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  display: inline-block;
}

.emotion-0:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<a
    class="emotion-0"
    href="/"
  />
</DocumentFragment>
`;
